app/components/post_draft/draft_input/index.tsx

```diff
@@ -19,6 +20,11 @@ type Props = {
     channelId: string;
     rootId?: string;
     currentUserId: string;
+    location: string;
```

Comment:
is this referring to source screen? if it is, I think we should change this prop name to be `sourceScreen` instead, here and in any other component that uses something like it.